WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -207,7 +204,18 @@ final public class PushNotificationsManager: NSObject {
     ///
     /// - Parameter userInfo: The Notification's Payload
     ///
-    func trackNotification(with userInfo: NSDictionary) {
+    func trackNotification(with userInfo: NSDictionary, response: UNNotificationResponse? = nil) {
+        let event: WPAnalyticsStat? = {
+            if let response {
+                return response.actionIdentifier == UNNotificationDefaultActionIdentifier ? .pushNotificationAlertPressed : nil
+            }
+            return .pushNotificationReceived
+        }()
```

Comment:
This logic fixes the bug where the event `push_notification_alert_tapped` is tracked when the app is in foreground. That event is only tracked when the user actually taps the notification alert.